coffee/directives/fixtable.coffee

```diff
@@ -48,26 +48,34 @@ angular.module 'fixtable'
 				return unless newVal?
 				getPageData()
 
-			# refresh when paging options change
-			scope.$watch 'options.pagingOptions', (newVal, oldVal) ->
-				return unless newVal
+			updatePagingOptions = (newVal, oldVal) ->
+				unless newVal then return 
+				pageTypeChanged = newVal.type isnt oldVal.type
+				
 				newVal.currentPage = parseInt newVal.currentPage
-				scope.totalPages = Math.ceil(newVal.totalItems / newVal.pageSize) or 1
-				scope.totalPagesOoM = (scope.totalPages+"").length
+				unless newVal.type is 'prevNext'
+					scope.totalPages = Math.ceil(newVal.totalItems / newVal.pageSize) or 1
+					scope.totalPagesOoM = (scope.totalPages+"").length
 
 				# don't allow currentPage to be set too high
 				if newVal.currentPage > scope.totalPages
 					newVal.currentPage = scope.totalPages
 
 				# run callback (on pagingOptions init or currentPage/pageSize change)
 				pageChanged = newVal.currentPage isnt oldVal.currentPage
+				
 				pageSizeChanged = newVal.pageSize isnt oldVal.pageSize
 				if pageSizeChanged
 					scope.options.pagingOptions.currentPage = 1
 
-				if newVal is oldVal or pageChanged or pageSizeChanged
+				if newVal is oldVal or pageChanged or pageSizeChanged or pageTypeChanged
 					getPageData()
 
+			# refresh when paging options change
+			scope.$watch 'options.pagingOptions', (newVal, oldVal) ->
```

Comment:
- [ ] Add a default to fixtableDefaultOptions, also a comment to document reasonable values for all the object props